introduction-to-datasets.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+The default storage _scheme_ (as opposed to a _[schema](#schemas)_, described below) is Hive. If you omit the scheme in your URI, your dataset is stored in Hive. For example, the URI `movies` is equivalent to `dataset:hive:/movies`. A dataset you create with this URI is stored as a table in Hive's `default` database.
```

Comment:
There isn't a default scheme in the API, so we shouldn't mention one in the overview. This is how the CLI behaves because we used to identify datasets by name instead of by the whole URI. We kept support for dataset names so old invocations would still work, which means that it seems like there is a default URI pattern.